db/schema.rb

```diff
@@ -136,18 +136,18 @@
     t.index ["uploader_id"], name: "index_shf_documents_on_uploader_id", using: :btree
   end
 
-  create_table "uploaded_files", force: :cascade do |t|
+  create_table "uploaded_files", id: :bigserial, force: :cascade do |t|
     t.datetime "created_at",                null: false
     t.datetime "updated_at",                null: false
     t.string   "actual_file_file_name"
     t.string   "actual_file_content_type"
     t.integer  "actual_file_file_size"
     t.datetime "actual_file_updated_at"
-    t.integer  "membership_application_id"
+    t.bigint   "membership_application_id"
     t.index ["membership_application_id"], name: "index_uploaded_files_on_membership_application_id", using: :btree
   end
 
-  create_table "users", force: :cascade do |t|
+  create_table "users", id: :bigserial, force: :cascade do |t|
     t.string   "email",                  default: "",    null: false
     t.string   "encrypted_password",     default: "",    null: false
     t.string   "reset_password_token"
```

Comment:
I'm not sure how these differences occurred (did not change any migration file).   Hopefully they are innocuous ...